File: app/src/main/java/dev/studiocloud/instamovie/data/services/response/movieDetailResponse/GenresItem.java
package dev.studiocloud.instamovie.data.services.response.movieDetailResponse;

import com.google.gson.annotations.SerializedName;

class GenresItem{
	@SerializedName("name")
	private String name;

	@SerializedName("id")
	private int id;

	public void setName(String name){
		this.name = name;
	}

	public String getName(){
		return name;
	}

	public void setId(int id){
		this.id = id;
	}

	public int getId(){
		return id;
	}

	@Override
	public String toString(){
		return
		"GenresItem{" +
				"name = '" + name + '\'' +
				",id = '" + id + '\'' +
				"}";
	}
}

File: app/src/main/java/dev/studiocloud/instamovie/data/services/response/tvDetailResponse/GenresItem.kt
package dev.studiocloud.instamovie.data.services.response.tvDetailResponse

import com.google.gson.annotations.SerializedName

class GenresItem {
    @SerializedName("name")
    var name: String? = null

    @SerializedName("id")
    var id = 0
    override fun toString(): String {
        return "GenresItem{" +
                "name = '" + name + '\'' +
                ",id = '" + id + '\'' +
                "}"
    }
}